docs/EN/resources/FAQ.md

```diff
@@ -0,0 +1,195 @@
+
+### I've turned on SMBs, but every time I receive an SMB, Trio cuts back or zero-temps my basal. Is that normal?
+Yes. SMBs effectively borrow insulin from the future to front load insulin delivery to assist in avoiding glucose spikes. Long, low, or zero temporary basal rates are expected to be observed with SMBs activated. This allows the algorithm to balance out the peak insulin timing and avoid hypoglycemia secondary to the increased insulin on board. 
+
+### How do I make SMBs more responsive?
+SMBs are limited by your Max `Max SMB Basal Minutes` and `Max UAM SMB Basal Minutes`. The default for both of these settings is 30 minutes. This default is conservative for most users, and it is suggested that this value be increased while monitoring the effect gradually. Raising this value will allow for bigger SMBs, thus making the dosing stronger. You need to decide how much you want to limit (or not limit) your SMBs.
+
+### Why isn't Trio looping when CGM glucose is 40 mg/dl (2.2 mmol/l) or lower?
+Trio can’t determine the glucose value when under 40mg/dL (2.2 mmol/L). This is why the CGM apps from Dexcom and Libre are displaying “Low.” A “Low” reading is also sometimes due to a sensor error or a faulty sensor needing replacement. Trio can't safely make a glucose prediction without knowing the glucose value or if it’s rising or falling.
+
+### Why isn't Trio looping or delivering insulin when CGM glucose is 400 mg/dl (22 mmol/l) or higher?
+Trio can’t determine the glucose value when over 400mg/dL (22 mmol/L). This is why the CGM apps from Dexcom and Libre display “High.” A “High” reading is also sometimes due to a sensor error or a faulty sensor needing replacement. Trio can't safely make a glucose prediction without knowing the glucose value or if it’s rising or falling.
+<br>
+
+## Omnipod FAQs
+
+### What pods work with Trio?
+You can use DASH and Eros Omnipod pods with the Loop app. You cannot use Omnipod 5 pods. 
+
+### Do I still need a PDM with Trio?
+You do not need the Omnipod Personal Diabetes Manager (PDM); you need the pod supplies. They will pair with only one device at a time. So, a pod is either paired with a PDM or your Trio app on your iPhone. 
+
+In other words, your PDM can stay in the diabetes closet while you are Looping. You cannot use the PDM for a pod activated with the Trio app. That doesn't mean you should get rid of your PDM if you have one. Instead, keep it for backup situations if you lose your phone. 
+
+### Does Trio cause an increase in pod failures?
+There is more communication between your phone and the pod than the PDM. This increases the load on the pod battery. Most people have no increase in pod failures. However, it does occasionally happen. Below are some tips to take to limit battery drain:
+* Have a correction range instead of a single number
```

Comment:
This is only applicable to Loop, not Trio. Can be removed from FAQs.

Reply:
Fixed